support/e2e/package.json

```diff
@@ -24,5 +25,9 @@
     "playwright-testing-library": "^2.7.0",
     "signal-exit": "^3.0.3",
     "test-keyboard": "1.0.0-next.49"
+  },
+  "devDependencies": {
+    "@testim/root-cause-core": "^0.1.23-canary-540.2",
```

Comment:
I don't know pnpm very well, but
`@testim/root-cause-core` is a dependency of `@testim/root-cause-jest`,
and also contains the cli tool to view the results.

With yarn & npm, if i install `@testim/root-cause-jest` i get also the cli tool linked to the local `.bin`,
but here without explicitly installing `@testim/root-cause-core` the cli was missing

Reply:
By default, `pnpm` doesn't hoist dependencies. I've left it that way since it helped prevent a bunch of bugs I used to run into when publishing packages. 

If you want to install the package at the root of the project then you can run it with:

```bash
pnpm run root-cause -W
```

Reply:
We've fixed that issue, and:
`pnpx root-cause` will run root cause cli